app/modules/app_ui.py

```diff
@@ -44,10 +43,21 @@ def init_app(app):
     app.register_blueprint(backend_blueprint)
 
 
+@frontend_blueprint.route('/', endpoint='root')
+@frontend_blueprint.route('/auth/code/<string:code>', endpoint='auth-code')
+@frontend_blueprint.route(
+    '/pending-sightings/<string:guid>', endpoint='pending-sightings'
+)
+@frontend_blueprint.route('/sightings/<string:guid>', endpoint='sightings')
+def dummy_view(*args, **kwargs):
+    """This view is created just for frontend route generation"""
+    return ''
```

Comment:
I would raise an explicit `RuntimeError` or `NotImplementedError` here, we want it to be clear that this code should never run

Reply:
Ah ok, I can do that